torch/_functorch/_activation_checkpointing/knapsack.py

```diff
@@ -0,0 +1,125 @@
+# mypy: ignore-errors
```

Comment:
Why are we ignoring all mypy errors here?

Reply:
oh, just copying the what other files in the directory where doing...will remove